tensorflow/tools/ci_build/Dockerfile.cpu

```diff
@@ -8,6 +8,7 @@ RUN /install/install_bootstrap_deb_packages.sh
 RUN add-apt-repository -y ppa:openjdk-r/ppa
 RUN /install/install_deb_packages.sh
 RUN /install/install_bazel.sh
+RUN /install/install_pip.sh
```

Comment:
No need for this line once https://github.com/tensorflow/tensorflow/pull/1472 is in.